Netflix Clone-2/index.html

```diff
@@ -1,156 +0,0 @@
-<html>
-
-<head>
-    <meta charset="utf-8" />
```

Comment:
update ur branch 
u r deleting this file

Reply:
Done with changes